dotcom-rendering/docs/architecture/3rd-party-technical-review/002-ipsos-mori.md

```diff
@@ -91,7 +91,7 @@ Similar security and data privacy considerations are applicable also for AMP. Th
 
 ## Perfomance engineering review
 
-Adding another tracker sending regularly will have a negative impact on performance. The Dotmetrics script on web loads additional scripts which total around 31kB in an initial experiment. The main script loaded in particular is 28kB on its own, which for comparison is larger than google analytics and 15x the size of comscore.
+Adding another tracker sending regularly will have a negative impact on performance. The Dotmetrics script on web loads additional scripts which total around 31kB in an initial experiment. The main script loaded in particular is 28kB on its own, which for comparison is 15x the size of comscore.
```

Comment:
I would keep this paragraph as-is for now, because it makes references to other deprecated tracking tools, but the general idea still remains: there is a cost to adding too many scripts!

Reply:
Added back 89f9b9d3582287a764547f9e2b01873025a3fd45. Do we really need references to deprecated tools? Shouldn't this paragraph be deleted at some point?

Reply:
In this case @deedeeh I believe because the are specifically historical 'architectural decision records' we don't modify them as they are a record of decisions made at that time and not 'living' documentation.

Reply:
Ah Ok! That makes sense. We don't touch the 'architectural decision records' got it 👍🏼